README.NASHORN.POLYFILLS.md

```diff
@@ -1,39 +1,15 @@
 # react4xp-runtime-nashornpolyfills
 
-Supplies a JS file intended to run through the react4xp server-side rendering engine, polyfilling nashorn with a set of
-JS functionality needed to render react on the server side. Also supplies a webpack config for compiling this JS file to
-the location expected by the react4xp runtime.
+Supplies a webpack config for compiling a JS file intended to run through the
+react4xp server-side rendering engine, polyfilling nashorn with a set of JS
+functionality needed to render react on the server side.
 
-Groundwork is laid here for developers to add more nashorn polyfilling code, which should automatically be added to the
-SSR engine. This needs more testing, might work or not - for now, regard it as a future feature.
+Groundwork is laid here for developers to add more nashorn polyfilling code,
+which should automatically be added to the SSR engine. This needs more testing,
+might work or not - for now, regard it as a future feature.
```

Comment:
I guess we're testing and implementing it for real now, so this last sentence can be removed